apps/src/music/views/MusicView.jsx

```diff
@@ -153,6 +176,42 @@ class UnconnectedMusicView extends React.Component {
         currentPlayheadPosition: this.player.getCurrentPlayheadPosition()
       });
     }
+
+    this.progressManager?.updateProgress();
```

Comment:
I'll adjust it so that it only checks while we are playing, but we need it to check frequently while playing, because the current success criteria are for things actually happening in realtime while the program is running (like, are three sounds playing simultaneously).  We could evaluate them in other ways, like examining the program, or examining the timeline, but for tasks such as "trigger a sound using the Beatpad" we'd need to be doing this kind of realtime checking anyway.  This is the approach we took for validation in Star Wars & AI for Oceans, too.